src/daemon/rpc.rs

```diff
@@ -73,9 +81,307 @@ impl rpc::VarlinkInterface for Server {
             Err(e) => call.reply_invalid_parameter(e),
         }
     }
+
+    fn monitor(&self, call: &mut dyn rpc::Call_Monitor) -> varlink::Result<()> {
+        debug!("starting monitor worker loop");
```

Comment:
This debug message seems likely to cause spam. It's probably useful for building this feature, but I suggest removing it before this PR is merged.